nixos/modules/services/networking/yggdrasil.nix

```diff
@@ -121,6 +121,13 @@ in
         restarted. Keys are stored at ${keysPath}.
       '');
 
+      extraArgs = mkOption {
+        type = listOf str;
+        default = [ ];
+        example = [ "-loglevel info" ];
```

Comment:
Is this correct after `escapeShellA` is used?  Should it be `[ "-loglevel" "info" ]` like the one used in the nixos test?